Engine/Filesystem/PathAtlas.h

```diff
@@ -40,6 +40,7 @@
 #define RESOURCES_NAVMESH_PATH "/Resources/NavMesh"
 
 #define RESOURCES_BUILD_OPTIONS "/Resources/Scenes/build.options"
+#define PROJECT_TAGS_PATH "/Assets/ProjectSettings/tags.options"
```

Comment:
As Enrique suggested this should be moved because in Game we don't have this folder. (This can be together with the game.input in a ProjectSetting Folder inside Game Folder in the Game Repo